packages/list-store/src/search-params.test.ts

```diff
@@ -82,8 +131,111 @@ describe('getUrlWithSearchParams', () => {
         genres: ['filter5'],
       },
     };
+
     expect(getUrlWithSearchParams(options)).toBe(
       '/?query=my+query&offset=20&sortBy=nameDesc&licenses=filter1%2Cfilter2&publishers=filter3&spatialCoverages=filter4&genres=filter5'
     );
   });
 });
+describe('fromSearchParamsToSearchOptions', () => {
+  it('returns default search options if there are no search params', () => {
+    expect(
+      fromSearchParamsToSearchOptions({options, searchParams: {}})
+    ).toStrictEqual({
+      filters: {},
+      offset: 0,
+      sortBy: 'relevance',
+      sortOrder: 'desc',
+      limit: 10,
+      query: undefined,
+    });
+  });
+
+  it('returns default search options if the search params contain defaults', () => {
+    const searchParams = {
+      offset: '0',
+      sortBy: defaultSortBy,
+    };
+
+    expect(
+      fromSearchParamsToSearchOptions({options, searchParams})
+    ).toStrictEqual({
+      filters: {},
+      offset: 0,
+      sortBy: 'relevance',
+      sortOrder: 'desc',
+      limit: 10,
+      query: undefined,
+    });
+  });
+
+  it('returns default search options if there are invalid search params', () => {
+    const searchParams = {
+      offset: 'string',
+      sortBy: 'color',
+    };
+
+    expect(
+      fromSearchParamsToSearchOptions({options, searchParams})
+    ).toStrictEqual({
+      filters: {},
+      offset: 0,
+      sortBy: 'relevance',
+      sortOrder: 'desc',
+      limit: 10,
+      query: undefined,
+    });
+  });
+
+  it('converts valid search params to search options', () => {
+    const searchParams = {
+      query: 'My query',
+      offset: '10',
+      sortBy: SortBy.NameAsc,
+      owners: 'owner1,owner2',
+      types: 'type',
+      subjects: 'subject',
+    };
+
+    expect(
+      fromSearchParamsToSearchOptions({options, searchParams})
+    ).toStrictEqual({
+      query: 'My query',
+      filters: {
+        owners: ['owner1', 'owner2'],
+        types: ['type'],
+        subjects: ['subject'],
+      },
+      offset: 10,
+      sortBy: 'name',
+      sortOrder: 'asc',
+      limit: 10,
+    });
+  });
+});
+
+describe('getClientSortBy', () => {
+  it('finds the client SortBy', () => {
```

Comment:
Minor: the word 'finds' suggests this is a search action, but the name of the function is 'get...'. Harmonize? For example: `it('gets the...')` or `it('returns the...')`